OpenDreamShared/Dream/Procs/ProcAttributes.cs

```diff
@@ -6,7 +6,7 @@ namespace OpenDreamShared.Dream.Procs;
 [Flags]
 public enum ProcAttributes
 {
-    None = 0, // Internal
+    None = 1 << 0, // Internal
```

Comment:
Why was this changed to 1?

Reply:
Because it made the default value setting serialize it properly.